bin/swagger_style.py

```diff
@@ -43,13 +47,23 @@ def main() -> int:
     result = swagger_to.style.perform(swagger=swagger, typedefs=intermediate_typedefs, endpoints=endpoints)
 
     if result:
+        if args.with_line_number:
+            result.sort(key=lambda complaint: complaint.line)
+
         print("Style checks failed: \n")
-        if args.verbose:
-            for cmpl in result:
-                print("{}: {}: \"{}\"".format(cmpl.where, cmpl.message, cmpl.what.replace('\n', ' ')))
-        else:
-            for cmpl in result:
-                print("{}: {}".format(cmpl.where, cmpl.message))
+        for complaint in result:
```

Comment:
I'd put refactor this snippet into a separate function (`format_complaint` or `generate_complaint`?)